apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -424,14 +424,6 @@ function load_block_description_links() {
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_block_description_links' );
 
-/**
- * Load Tutorials API
- */
-function load_tutorials() {
-	require_once __DIR__ . '/tutorials/tutorials.php';
-}
-add_action( 'plugins_loaded', __NAMESPACE__ . '\load_tutorials' );
```

Comment:
Do we need to separate this into 2 PRs to deploy these in stages (removing files vs. references to them separately)? Or is that something you plan on handling during the wpcom deploy itself?

Reply:
Yeah that gets handled during wpcom deploy. I'll create one Diff to just do this part (don't load) and then another to remove the actual files.